sys/include/net/gnrc/netdev/lora_adpt.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Please call this file `sys/include/net/gnrc/netdev/lora.h` to be more in line with `gnrc/netdev/ieee802154.h` and `gnrc/netdev/eth.h`. `xbee_adpt.h` is a special case, since it is IEEE 802.15.4 but uses its own communication protocol to interact with the device.